node/setup.go

```diff
@@ -131,8 +131,14 @@ func logNodeStartupInfo(state sm.State, pubKey crypto.PubKey, logger log.Logger,
 	}
 }
 
-func onlyValidatorIsUs(state sm.State, pubKey crypto.PubKey) bool {
+func hasMajorityVotingPower(state sm.State, pubKey crypto.PubKey) bool {
 	if state.Validators.Size() > 1 {
+		for _, val := range state.Validators.Validators {
+			if val.VotingPower > state.Validators.TotalVotingPower()*2/3 {
```

Comment:
We also need to check that this validator is us i.e. `pubKey != nil && bytes.Equal(pubKey.Address(), addr)`

Reply:
Does it matter whether it is us or another validator that has majority when in any case we should not have a network where only one guy has majority?

Reply:
It's okay to start a network with a single node that has majority. Many testnets and sometimes mainnets start this way before the power becomes more evenly distributed